src/server/views/help/what-happens-next/apply-for-a-county-parish-holding-cph-number.html

```diff
@@ -0,0 +1,5 @@
+<h2 class="govuk-heading-m">What happens next</h2>
+
+<p class="govuk-body">RPA will send you an email with a validation link to confirm your email address. You must click on this link within 12 hours. If you do not do this within 12 hours, call RPA on 03000 200 301 to get another email validation link.</p>
```

Comment:
One to check, but I didn't know what RPA stands for

From the content and publishing style guide entry for [**Abbreviations and acronyms**](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#abbreviations-and-acronyms):

>The first time you use an abbreviation or acronym explain it in full on each page unless it’s well known, like UK, DVLA, US, EU, VAT and MP. This includes government departments or schemes. Then refer to it by initials, and use [acronym Markdown](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#acronyms) so the full explanation is available as hover text.
>
>If you think an acronym is well known, please provide evidence that 80% of the UK population will understand and commonly use it. Evidence can be from search analytics or testing of a representative sample.

Can we follow the usual advice to expand this acronym out?

E.g. The [**Privacy Notice** page](https://www.gov.uk/guidance/rpa-privacy-notices) says "Rural Payments Agency (RPA)" initially

Reply:
Steve approved, Done.